fulp_modules/features/halloween/2021/ace_attorney/bailey_reaper.dm

```diff
@@ -0,0 +1,34 @@
+/**
+ * Barok Van Zieks
+ * From: Ace Attorney Chronicles
+ * By: Pepsilawn
+ */
+
+/obj/item/clothing/neck/costume_2021/breaper
```

Comment:
can you name these breaper_cape and such so it doesnt get confused with them all being named the same